src/pipeline.js

```diff
@@ -90,53 +88,51 @@ class Pipeline {
 
     // function chain that was defined last. used for `when` and `unless`
     this._last = null;
```

Comment:
I think the `this._last` is not used anymore

Reply:
Yes it is used in `when()`

Reply:
Ah, you mean I could use `this._steps` directly there, now that we only have one sequence of functions... good point!

Reply:
No, we still have 2 queues: `_steps` and `_taps` (used by `pipe.every()`)